.travis.yml

```diff
@@ -44,7 +44,7 @@ jobs:
     - stage: whitesource
       script: git branch -f "$TRAVIS_BRANCH" && git checkout "$TRAVIS_BRANCH" && sbt -jvm-opts .jvmopts-travis whitesourceCheckPolicies whitesourceUpdate
     - stage: publish
-      script: sbt -jvm-opts .jvmopts-travis ++2.12.10 publish ++2.13.1 "project akka-grpc-maven-plugin; set publish/skip := true; project akka-grpc-codegen; set publish/skip := true; project akka-grpc-scalapb-protoc-plugin; set publish/skip := true; project sbt-akka-grpc; set publish/skip := true; project akka-grpc" publish
+      script: sbt -jvm-opts .jvmopts-travis ++2.12.10 publish ++2.13.1 akka-grpc-runtime/publish ++2.13.1! akka-grpc-codegen/publish
```

Comment:
Ah!

There's a sneaky `!` to force publishing the `2.13` artifact of `-codegen`